src/core/public/i18n/i18n_service.tsx

```diff
@@ -23,16 +23,209 @@ import { EuiContext } from '@elastic/eui';
+        }),
+      'euiBottomBar.screenReaderAnnouncement': i18n.translate(
+        'core.euiBottomBar.screenReaderAnnouncement',
+        {
+          defaultMessage:
+            'There is a new menu opening with page level controls at the end of the document.',
+        }
+      ),
+      'euiCodeBlock.copyButton': i18n.translate('core.euiCodeBlock.copyButton', {
+        defaultMessage: 'Copy',
+      }),
+      'euiCodeEditor.startEditing': i18n.translate('core.euiCodeEditor.startEditing', {
+        defaultMessage: 'Press Enter to start editing.',
+      }),
+      'euiCodeEditor.startInteracting': i18n.translate('core.euiCodeEditor.startInteracting', {
+        defaultMessage: 'Press Enter to start interacting with the code.',
+      }),
+      'euiCodeEditor.stopEditing': i18n.translate('core.euiCodeEditor.stopEditing', {
+        defaultMessage: "When you're done, press Escape to stop editing.",
+      }),
+      'euiCodeEditor.stopInteracting': i18n.translate('core.euiCodeEditor.stopInteracting', {
+        defaultMessage: "When you're done, press Escape to stop interacting with the code.",
+      }),
+      'euiCollapsedItemActions.allActions': i18n.translate(
+        'core.euiCollapsedItemActions.allActions',
+        {
+          defaultMessage: 'All actions',
+        }
+      ),
+      'euiColorPicker.colorSelectionLabel': ({ colorValue }: EuiValues) =>
+        i18n.translate('core.euiColorPicker.colorSelectionLabel', {
+          defaultMessage: 'Color selection is {colorValue}',
+          values: { colorValue },
+        }),
+      'euiColorPicker.transparentColor': i18n.translate('core.euiColorPicker.transparentColor', {
+        defaultMessage: 'transparent',
+      }),
+      'euiComboBoxOptionsList.allOptionsSelected': i18n.translate(
+        'core.euiComboBoxOptionsList.allOptionsSelected',
+        {
+          defaultMessage: "You've selected all available options",
+        }
+      ),
+      'euiComboBoxOptionsList.alreadyAdded': ({ label }: EuiValues) =>
+        i18n.translate('core.euiComboBoxOptionsList.alreadyAdded', {
+          defaultMessage: '{label} has already been added',
+          values: { label },
+        }),
+      'euiComboBoxOptionsList.createCustomOption': ({ key, searchValue }: EuiValues) =>
+        i18n.translate('core.euiComboBoxOptionsList.createCustomOption', {
+          defaultMessage: 'Hit {key} to add {searchValue} as a custom option',
+          values: { key, searchValue },
+        }),
+      'euiComboBoxOptionsList.loadingOptions': i18n.translate(
+        'core.euiComboBoxOptionsList.loadingOptions',
+        {
+          defaultMessage: 'Loading options',
+        }
+      ),
+      'euiComboBoxOptionsList.noAvailableOptions': i18n.translate(
+        'core.euiComboBoxOptionsList.noAvailableOptions',
+        {
+          defaultMessage: "There aren't any options available",
+        }
+      ),
+      'euiComboBoxOptionsList.noMatchingOptions': ({ searchValue }: EuiValues) =>
```

Comment:
__issue:__ `searchValue` is a React element and function should return React element as well, right? If so you need to use `FormattedMessage` instead:
```tsx
      'euiComboBoxOptionsList.noMatchingOptions': ({ searchValue }: EuiValues) => (
        <FormattedMessage
          id="core.euiComboBoxOptionsList.noMatchingOptions"
          defaultMessage="{searchValue} doesn't match any options"
          values={{ searchValue }}
        />
      ),
```